src/Json.Pointer/README.md

```diff
@@ -0,0 +1,221 @@
+# Microsoft.Json.Pointer
```

Comment:
># [](start = 0, length = 1)

To see the formatted result, visit [https://github.com/Microsoft/jschema/blob/users/lgolding/41-document-json-pointer/src/Json.Pointer/README.md](https://github.com/Microsoft/jschema/blob/users/lgolding/41-document-json-pointer/src/Json.Pointer/README.md).